.appveyor.yml

```diff
@@ -74,11 +74,23 @@ after_test:
 cache:
   - '%R_LIB_PATH%'
 
+on_failure:
+  - 7z a failure.zip *.Rcheck\*
+  - appveyor PushArtifact failure.zip
+
 artifacts:
   - path: python-package\dist\*
     name: pip
   - path: python-package\compile\RGF\bin\*
     name: rgf_exe
+  - path: '*.Rcheck\**\*.log'
+    name: Logs
+  - path: '*.Rcheck\**\*.out'
+    name: Logs
+  - path: '*.Rcheck\**\*.fail'
+    name: Logs
+  - path: '*.Rcheck\**\*.Rout'
+    name: Logs
```

Comment:
Do we really need all these files in case of successful run? I think it's enough to have `failure.zip`  in `on_failure` section.

Reply:
If you don't mind I would suggest that we keep the *path-name* statements till the issue is resolved and then we can remove *.log* and *.out* paths. The *.Rout* and *.fail* should be kept in any case as they are related with the output of the *tests* and *examples* of the R-package.

Reply:
OK, I don't mind.